src/foremast/consts.py

```diff
@@ -155,7 +155,7 @@ def _convert_string_to_native(value):
     result = None
 
     try:
-        result = ast.literal_eval(value)
+        result = ast.literal_eval(str(value))
```

Comment:
I'm wondering whats the value that got passed that requires this.

Reply:
Sharing payload privately due to potential sensitivity.

Reply:
No issue here after reviewing the scrubbed config.